.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,67 @@
+# Slimefun4 - Code of Conduct
+### What is a Code of Conduct?
+> A code of conduct is a document that establishes expectations for behavior for your project’s participants. 
+> Adopting, and enforcing, a code of conduct can help create a positive social atmosphere for your community.
+<br>\- *Source: [https://opensource.guide](https://opensource.guide/code-of-conduct/#why-do-i-need-a-code-of-conduct)*
+
+This document should serve the purpose of outlining the behaviour we expect from any participant of the project.<br>
+
+## Scope
+This Code of Conduct applies to all sections of the [Slimefun4 GitHub repository](https://github.com/TheBusyBiscuit/Slimefun4), our [Slimefun GitHub organization](https://github.com/Slimefun) and all repositories owned by said organization.<br>
+For our official discord server, please refer to our article on [Discord Rules](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Discord-Rules).
+
+Everyone who engages with this project on any of these repositories is expected to follow the Code of Conduct.<br>
+This includes maintainers, contributors, sponsors and anyone who engages in the "Issues" section on GitHub.
+
+## Our Standards
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
```

Comment:
*towards

Reply:
I knew it. But it's the standard text so I didn't dare to change it.